### ISSUE TITLE
Unable to get participants' info as a competition organizer

### ISSUE THREAD

**Issue by richardaecn** [codalab/codalab-competitions#2370]
Hi,

I am the organizer for COCO Detection Challenge: https://competitions.codalab.org/competitions/5181 . We are working on deciding the challenge winners for our workshop at ECCV and finalizing the schedule for invited speakers. However, I am unable to get participants' info for the challenge submissions. I tried the following and both of them failed:
1) Click "Participants" tab in Admin features, it's always in the loading status and will finally gives timeout error. 
2) Click "Download all submissions on leaderboard" under Results -> test-challenge2018 (segm) phase and it always gives the error message: "There was an error retrieving the file. Please try again later or report the issue.". 

@FlavioAlexander

**Comment by richardaecn** [comment]
While fixing this issue, is it possible to send me the list of all participants' email addresses for https://competitions.codalab.org/competitions/5181? Thanks for your help!

**Comment by richardaecn** [comment]
@ckcollab

**Comment by ckcollab** [comment]
> While fixing this issue, is it possible to send me the list of all participants' email addresses

I don't think we can share this information, although it would be a quick fix :(

We are looking into this strange problem and trying to find a solution, will let you know when we have new information for you.

**Comment by richardaecn** [comment]
Thanks!

**Comment by Tthomas63** [comment]
Hello @richardaecn,

Sorry for the inconvenience. Looking into this we think we found the cause of the issue and we should have a fix for you soon.

Best regards,
Tyler

**Comment by richardaecn** [comment]
Great. Thanks for the update!

**Comment by Tthomas63** [comment]
Hello @richardaecn,

We pushed a temporary fix addressing this issue which fixes the page not loading. Please let us know if everything looks alright

Best regards,
Tyler

**Comment by richardaecn** [comment]
Hi @Tthomas63 ,

Thanks for your help! I am able to get participants' info now.
